pipelines/zap/pipeline.yml

```diff
@@ -74,14 +67,31 @@ jobs:
   - get: scripts
   - get: project-data
     resource: fetch-project-data-<%= project['name'] %>
-  - get: s3-bucket-results
+  # if it's a new project, the last results won't exist
+  - try:
+      get: last-results
+      resource: s3-result-<%= project['name'] %>
+      on_failure:
+        task: create-last-results
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: busybox
+          outputs:
+          - name: last-results
+          run:
+            path: echo
+            args:
+            - Creating empty directory for results from the last scan.
```

Comment:
This is a bit of a hack, but it works. We will just need to watch out for our scans reporting "NEW SITE" over and over, which means there's something wrong fetching the files.